culpa.csl

```diff
@@ -0,0 +1,1834 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" page-range-format="expanded" demote-non-dropping-particle="never">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>CULPA (Copenhagen University Legal Propper Attribution)</title>
+    <title-short>CULPA </title-short>
+    <id>http://www.zotero.org/styles/culpa</id>
+    <link rel="self" href="http://www.zotero.org/styles/culpa"/>
```

Comment:
Please add a link with `documentation` to where this style is documented and also a link with `template` for the style you used as a template.